src/platform/Kconfig

```diff
@@ -256,6 +259,13 @@ config IMX
 	  This has to be selected for every i.MX NXP platform.
 	  It enables NXP platforms-specific features.
 
+config IMX_INTERRUPT_IRQSTEER
+	bool
+	default n
+	help
+	  This has to be selected for i.MX NXP platform that have irqsteer.
+	  It enables NXP platforms-specific features.
+
```

Comment:
@TangleZ please move this config definition to src/drivers/imx/Kconfig.

Other than that, looks good to me.